resources/views/common/generic-reuse-item.blade.php

```diff
@@ -12,10 +12,22 @@
                     @if(($item->getStatus() != "approved" && $item->user_id === Auth::user()->id) || Auth::user()->is_admin)
                         <div class="my-3 row d-flex justify-content-between">
                             <span class="text-center col-md-4">
-                                <a href="{{route('submissionEdit', ["type" => $item->getLocationType(), "state" => $item->getStatus(), "itemId" => $item->id, "back" => url()->full(), "previousBack" => $backUrl])}}" class="btn btn-primary btn-block"> Edit </a>
+                                <a href="{{route('submissionEdit', ["type" => $item->getLocationType(), "state" => $item->getStatus(), "itemId" => $item->id, "back" => url()->full(), "previousBack" => $backUrl])}}"
+                                   class="btn btn-primary btn-block"> Edit </a>
                             </span>
                             <span class="text-center col-md-4">
-                            <button class="btn btn-danger btn-block" data-toggle="modal" data-target="#exampleModal" type="button">Delete</button>
+                                <form method="POST" @if($type==="state") action="{{route('addStateMergeSubmit')}}"
+                                      @elseif($type==="county") action="{{route('addCountyMergeSubmit')}}"
+                                      @else action="{{route('addCityMergeSubmit')}}"@endif>
+                                                                   {{ csrf_field() }}
+                                                             <input name="id"
+                                                                    value="{{$item->id}}" hidden>
+                                <button type="submit" class="btn btn-success btn-block">Approve</button>
+                                </form>
```

Comment:
I like it, but this also shows up when viewing already approved submissions... Could you call `$item->getStatus()` and check if it is equal to `approved`, and if so, don't display the approve button.